people-details-sql-jdbc/src/main/java/dao/PersonDAOInterface.java

```diff
@@ -0,0 +1,14 @@
+package dao;
+
+public interface PersonDAOInterface<T> {
+
+    void findAll();
+
+    void create(T type);
+
+    void findById(int id);
```

Comment:
void? you should return `Optional<Person>` here - otherwise, how i am gonna use your api?